openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -572,6 +572,8 @@ def delete_users(users):
     for user in users:
         retire_user_comments(user)
 
-    delete_profile_images.delay(users)
+    # Delete user profile images on background task
```

Comment:
_Nit_: wouldn't it be better to use `in the` instead of `on`?

Reply:
Yes, I'll fix it and merge once the tests pass. :+1: